pkgs/by-name/ov/overskride/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, fetchFromGitHub, stdenv, rustPlatform, cargo, rustc, meson, ninja
+, pkg-config, wrapGAppsHook4, desktop-file-utils, appstream-glib
+, blueprint-compiler, dbus, gtk4, libadwaita, bluez, libpulseaudio }:
+let
+
+  pname = "overskride";
+  version = "0.5.6";
+  owner = "kaii-lb";
+
+  src = fetchFromGitHub {
+    inherit owner;
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-syQzHHT0s15oj8Yl2vhgyXlPI8UxOqIXGDqFeUc/dJQ=";
+  };
+
+in stdenv.mkDerivation {
+
+  inherit src pname version;
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    name = "${pname}-${version}";
+    hash = "sha256-NEsqVfKZqXSLieRO0BvQGdggmXXYO15qVhbfgAFATPc=";
+  };
+
+  nativeBuildInputs = [
+    rustPlatform.cargoSetupHook
+    pkg-config
+    wrapGAppsHook4
+    desktop-file-utils
+    appstream-glib
+    blueprint-compiler
+    meson
+    ninja
+    cargo
+    rustc
+  ];
+
+  buildInputs = [ dbus gtk4 libadwaita bluez libpulseaudio ];
+
```

Comment:
```suggestion

  dontCargoBuild = true;

```

Reply:
you may also want dontCargoCheck

Reply:
Not necessary, as we have to override the `buildPhase` to make it work with meson.